lib/cli/commands/listswaps.ts

```diff
@@ -0,0 +1,93 @@
+import { callback, loadXudClient } from '../command';
+import { Arguments } from 'yargs';
+import { ListSwapsRequest, ListSwapsResponse } from '../../proto/xudrpc_pb';
+import Table , { HorizontalTable } from 'cli-table3';
+import colors from 'colors/safe';
+
+const FAILED_HEADERS = [
+  colors.blue('peer public key'),
+  colors.blue('order id'),
+  colors.blue('trading pair'),
+  colors.blue('quantity'),
+  colors.blue('failure reason'),
+];
+// TODO: create a function that generates these tables
+const SUCCESS_HEADERS = [
+  colors.blue('peer public key'),
+  colors.blue('order id'),
+  colors.blue('local id'),
+  colors.blue('trading pair'),
+  colors.blue('quantity'),
+  colors.blue('price'),
+  colors.blue('amount sent'),
+  colors.blue('amount received'),
+  colors.blue('currency received'),
+  colors.blue('currency send'),
+  colors.blue('role'),
+  colors.blue('rhash'),
```

Comment:
I think `Payment Hash` or maybe `Swap Hash` is more descriptive for users. Thoughts @kilrau ?

Reply:
I'd go for `Swap Hash` (since it's a hash for two payments, not one). Let's be consistent about using `Swap Hash` from now on.